courses/free-courses-en.md

```diff
@@ -675,6 +676,7 @@
 * [Data Processing and Feature Engineering with MATLAB](https://www.coursera.org/learn/feature-engineering-matlab) (coursera)
 * [Introduction to Matlab in English](https://www.youtube.com/playlist?list=PLGED90Y_uL1KLpdRmVtwfpNoYCWU9RPkK) -Mohammed Mohammed
 * [MATLAB and Simulink Training](https://matlabacademy.mathworks.com)
+* [MATLAB Programming for Numerical Computation NPTEL](https://www.youtube.com/playlist?list=PLRWKj4sFG7-6_Xr9yqg6SMr_F80KdFVhN) - Dr. Niket Kaisare NPTEL (YouTube)
```

Comment:
Authors name is not "Dr. Niket Kaisare NPTEL" . This repo mostly doesn't use honorifics like 'Dr.' or 'Prof.".

Reply:
could you please suggest the author name .... like would "Niket Kaisare NPTEL" do?